scripts/prepare-cluster.sh

```diff
@@ -1,15 +1,25 @@
 #!/bin/bash
 
-set -x
-
 function prepare() {
   echo "Preparing cluster..."
+  make -i helm-init helm-install-config spire-install
```

Comment:
Why `-i`?

Reply:
It's an attempt to add error detection in executing first two rules while preserving old workflow: all targets are executed regardless of errors in previous rules. 
In fact, it's a piece that was put before 'waiting for services' code, '-i' might be unnecessary now. Let try to remove it..